### PR TITLE
[JENKINS-57899] - fix saving global config

### PR DESCRIPTION
**Pull request by casz**
fixes JENKINS-57899

### CODE REVIEW

File: src/main/java/hudson/plugins/jira/JiraGlobalConfiguration.java [hunk 1]
```diff
@@ -29,5 +29,6 @@ public JiraGlobalConfiguration() {
     @DataBoundSetter
     public void setSites(List<JiraSite> sites) {
         this.sites = sites;
+        save();
```
Comment (casz) [inline src/main/java/hudson/plugins/jira/JiraGlobalConfiguration.java]: https://github.com/jenkinsci/jenkins/blob/01d7f443c9e3178da7222e1dff3fc8cf82bc6943/core/src/main/java/jenkins/model/GlobalConfiguration.java#L35
As noted by the java doc, you have to call save on the data-bound setters.

**Comment by olamy** [comment]
I'd really like to have unit test for that. Do you have time for that?

**Comment by olamy** [comment]
@casz if you can't no worries I will do it

**Comment by casz** [comment]
I'll add them shortly

**Comment by casz** [comment]
@olamy test added :)

**Comment by olamy** [comment]
@casz awesome! thanks!

**Comment by olamy** [comment]
@casz thanks a lot for the quick fix (and the test ;) )

**Pull request merged**